miasm2/arch/aarch64/sem.py

```diff
@@ -813,7 +835,6 @@ def __init__(self, symbol_pool=None):
         self.pc = PC
         self.sp = SP
         self.IRDst = m2_expr.ExprId('IRDst', 64)
-        self.addrsize = 64
```

Comment:
It seems you have removed a line you don't want to

Reply:
You are correct. I am sorry for my oversight.